### PR TITLE
Removed the material decoration in the DD4hep Plugin

### PR DESCRIPTION
**Pull request by Corentin-Allaire**
In the conversion from DD4hep geometry to the Acts Tracking geometry some volumes are decorated with material. This interfere with the material mapping and I believe it should be removed (since we don't want a one to one conversion of the material in the tracking geometry).

For now I have only commented out the corresponding line in case someone would want to keep them else I will remove them completely and edit this PR.

### CODE REVIEW

**Comment by msmk0** [comment]
I converted this to a draft to clarify that this can not be merged in right now.

**Comment by Corentin-Allaire** [comment]
Good idea thanks !

File: Plugins/DD4hep/src/DD4hepVolumeBuilder.cpp [hunk 1]
```diff
@@ -7,6 +7,9 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #include "Acts/Plugins/DD4hep/DD4hepVolumeBuilder.hpp"
+
+#include <boost/algorithm/string.hpp>
```
Comment (msmk0) [inline Plugins/DD4hep/src/DD4hepVolumeBuilder.cpp:11]: Is this include still needed? I think the only usage of `boost::iequals` is removed in this PR.

Comment (Corentin-Allaire) [reply Plugins/DD4hep/src/DD4hepVolumeBuilder.cpp:11]: Done

**Pull request merged**